tfw

```diff
@@ -1169,7 +1169,12 @@ __ensure_direct_lvm_setup() {
   local metadata_size="${2:-2G}"
 
   if [[ ! "${device}" ]]; then
-    device="$(awk '/active raid/ { print $1 }' </proc/mdstat)"
+    local device_basename
+    device_basename="$(awk '/active raid/ { print $1 }' </proc/mdstat)"
+
+    if [[ "${device_basename}" ]]; then
+      device="/dev/${device_basename}"
```

Comment:
In case there are multiple devices, this will yield interesting results. Maybe `{print $1; exit}` in the `awk` command, and be happy with the first that is found?